src/features/MoviesBrowser/ListPage/PersonTile/index.js

```diff
@@ -1,12 +1,26 @@
-import { Link } from "react-router-dom";
+
 import { toPerson } from "../../../../core/config/routes";
+import { StyledPersonTile, StyledLink, StyledPoster, PersonDetails } from "./styled";
 
-const PersonTile = ({ person }) => {
+const PersonTile = ({ person, showJob, showCharacter }) => {
+    
     return (
-        <div>
-            <img alt={person.name} src={`https://image.tmdb.org/t/p/w200${person.profile_path}`} />
-            <Link to={toPerson(person)}>{person.name}</Link>
-        </div>
+        <StyledPersonTile>
+            <StyledPoster
+            people={person.name} 
+            src={`https://image.tmdb.org/t/p/w200${person.profile_path}`}
+            />
+            <StyledLink
+            to={toPerson(person)}>{person.name}
+                        
+            </StyledLink>
+
+            <PersonDetails>
+            {showJob || <p> {person.job} </p>}
```

Comment:
Skoro całość jest w `<PersonDetails>` co jest optymalnym rozwiązaniem to w tych `showJob `i `showCharacter `trzeba usunąć `<p>` i `</p>`